### PR TITLE
feat: create the paste theme and make it the default

### PR DESCRIPTION
**Pull request by katieporter**
Create the Paste theme using the unified color palette, and make it the default theme for the Paste docs site

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [X] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: packages/paste-core/components/anchor/__tests__/anchor.test.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ describe('Anchor', () => {
   it('it should render an anchor', (): void => {
     const tree = renderer
       .create(
-        <Theme.Provider>
+        <Theme.Provider theme="console">
```
Comment (SiTaggart) [inline packages/paste-core/components/anchor/__tests__/anchor.test.tsx]: Prevents all the snapshots needing to be updated, plus it's good practice to set the theme

**Comment by SiTaggart** [comment]
As this is a large PR across multiple packages due to the type of change, once it's reviewed and OK to merge I'm going to redo all the git history to be package based, so that our changelogs are correct and clear.

File: packages/paste-website/src/components/site-wrapper/SiteBody.tsx [hunk 2]
```diff
@@ -19,3 +33,40 @@ export const SiteBody = styled.div`
...
+          paddingRight="space40"
+        >
+          <Text as="p" textAlign="center">
+            <strong>WARNING:</strong> The Paste theme is an <em>extremely early</em> preview of future work!{' '}
+            <Anchor
+              href="https://docs.google.com/document/d/1H2Rj3NEmVSv0yxMBRjYOjruQllO__uj2-I_ibIoumZs/edit?usp=sharing"
+              target="_blank"
+            >
+              Read the FAQ
+            </Anchor>{' '}
```
Comment (serifluous) [inline packages/paste-website/src/components/site-wrapper/SiteBody.tsx]: Just an FYI, Scott is working on an FAQ page for the Paste doc site (https://share.goabstract.com/0ee0c320-f4f8-483c-a56e-87fb301890c3?sha=59a451d9a07b0cd0162e4566dd63e244d1ff1838), and I've already asked him to mock it up with the unified design language FAQs. When we build this page, we can swap this link over 💪

Comment (katieporter) [reply packages/paste-website/src/components/site-wrapper/SiteBody.tsx]: @serifluous amazing, thank you so much for doing that!

**Pull request merged**